site/blog/cql-starter/cql-quickstart-cql.md

```diff
@@ -325,28 +328,27 @@ select * from starter.cqlstarter;
 
 You should see the single rampup entry along main operation entries in the Cassandra table.
 
-![alt_text](images/image2.png "image_tooltip")
 
-### Customize
+### 3. Customize
 
 Now, let’s customize the cql-starter to make it a bit more your own.
 
-1. Save the .yaml file to your local environment.
-2. Edit the file and uncomment under the default scenario the following entry:
+#### 1. Save the .yaml file to your local environment.
```

Comment:
Minor comment, we could specify the `--copy` option here. Maybe it's too much hand-holding as this is mentioned earlier in the `--list-workloads` section, but it could be helpful to make this easier.

Reply:
Good idea, I'll add that here!

Reply:
Gotcha, understand the suggestion now.  Thanks!